### DISCUSSION TITLE
Reset fader to zero with double click

### DISCUSSION THREAD

**Discussion by arcartis** [wwmm/easyeffects#1300] [Ideas]
Can we please have a "reset fader to zero" with a double click or Ctrl+click or something? That's really something that would be awesome.

**Comment by wwmm** [comment]
Fader? I am confused. We do not have a fader control as far as I remember.

**Comment by arcartis** [comment]
I mean these bad boys :) I don't know what you call them, but for me they're faders. Or maybe this is dependent on the individual plug-in? I'm not sure.
![image](https://user-images.githubusercontent.com/76786592/146642971-2e0af58e-aebe-4ad6-b726-b841772b52fc.png)

**Reply by wwmm** [reply]
They are gain controls.

**Reply by arcartis** [reply]
Okay, maybe I use different vocabulary, but that's essentially what I meant. It would be nice to have a way to reset them to zero without resetting all of the controls (faders) to zero, with the "Reset" button on the bottom. Just like most DAWs do. 

For example when using the EQ.
![image](https://user-images.githubusercontent.com/76786592/146644465-42cf6b35-0c1c-44ef-a156-b2907a8c2bbd.png)

Anyways, thanks for what you do. I love Easyeffects and I use it every day. 
Happy holidays and all the best for 2022.

**Reply by wwmm** [reply]
I will try to see if there is an easy way to implement that. Happy holidays! :-)
